server/lib/weather/weather.command.js

```diff
@@ -11,19 +37,55 @@ const { ServiceNotConfiguredError, NoValuesFoundError } = require('../../utils/c
  */
 async function command(message, classification, context) {
   let weather;
+  let weatherDay;
   try {
     const houses = await this.house.get();
     const house = houses[0];
     if (!house || !house.latitude || !house.longitude) {
       throw new NoValuesFoundError();
     }
+    weather = await this.get(house);
     switch (classification.intent) {
       case 'weather.get':
-        weather = await this.get(house);
         context.temperature = weather.temperature;
         context.units = weather.units === 'metric' ? '°C' : '°F';
+
         await this.messageManager.replyByIntent(message, `weather.get.success.${weather.weather}`, context);
         break;
+      case 'weather.tomorrow':
+        context.temperature_min = weather.days[0].temperature_min;
+        context.temperature_max = weather.days[0].temperature_max;
+        context.units = weather.units === 'metric' ? '°C' : '°F';
+        await this.messageManager.replyByIntent(
+          message,
+          `weather.tomorrow.success.${weather.days[0].weather}`,
+          context,
+        );
+        break;
+      case 'weather.after-tomorrow':
+        context.temperature_min = weather.days[1].temperature_min;
+        context.temperature_max = weather.days[1].temperature_max;
+        context.units = weather.units === 'metric' ? '°C' : '°F';
+        await this.messageManager.replyByIntent(
+          message,
+          `weather.after-tomorrow.success.${weather.days[1].weather}`,
+          context,
+        );
+        break;
+      case 'weather.day':
+        if (!context.day) {
+          throw new NotFoundError('day not found');
+        }
+        weatherDay = getWeatherByDay(weather, context.day);
+        if (!weatherDay) {
+          throw new NotFoundError('weather for this day not found');
```

Comment:
In commands, instead of throwing errors that will never be seen by the client, you should reply to the user with an error response

Reply:
I have add a new error type, try/catch and `replyByIntent` with a specific error key